src/Metadata/Property/PropertyMetadata.php

```diff
@@ -381,4 +383,29 @@ public function withSubresource(SubresourceMetadata $subresource = null): self
 
         return $metadata;
     }
+
+    /**
+     * Returns a new instance with the given alwaysIdentifier flag.
+     *
+     * @param bool $alwaysIdentifier
```

Comment:
You can remove both `@param` and `@return`, they are useless because of the typehints.